recipes/pcaone/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set version = "0.1.7" %}
+{% set sha256 = "e914dc2d3e1c64d027dd1d1b9338e4fc7789a647f7124cec717480dd0c584c25" %}
+
+
+package:
+  name: pcaone
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/Zilong-Li/PCAone/archive/refs/tags/v{{ version }}.tar.gz # [x86_64]
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+
+  host:
+    - mkl
+    - mkl-include
+    - zlib
+
+  run:
+    - mkl
+    - mkl-include
+    - zlib
+
+test:
+  commands:
+    - PCAone 
+
+about:
+  home: https://github.com/Zilong-Li/PCAone
+  license: GPL3
```

```suggestion
  license: GPL-3.0-only
```
Makes this a valid SPDX identifier.

---

done!